### PR TITLE
Add new logic for copy of Plugin and Bin directory

### PR DESCRIPTION
**Pull request by dragon512**
New copy logic should avoid root permission issue without forcing a copy of dtata
Add debugging code for Port issue we have been seeing

### CODE REVIEW

**Comment by dragon512** [comment]
@zwoop We need ss installed

**Comment by zwoop** [comment]
ss is installed...

[root@d766963623f8 /]# which ss
/usr/sbin/ss

**Comment by persiaAziz** [comment]
[approve ci autest]

**Comment by zwoop** [comment]
[approve ci autest]

**Comment by dragon512** [comment]
[approve ci]

File: tests/gold_tests/autest-site/trafficserver.test.ext [hunk 1]
```diff
@@ -42,26 +42,29 @@ def MakeATSProcess(obj, name, command='traffic_server', select_ports=True):
...
-    
+
     #ssl, storage & cache
     ssl_dir = os.path.join(ts_dir, 'ssl')
     storage_dir = os.path.join(ts_dir, 'storage')
     cache_dir = os.path.join(ts_dir, 'cache')
 
     # create process
     p = obj.Processes.Process(name, command)
+    p_debug = obj.Processes.Process("port-debug", "ss --listen --tcp --process")
```
Comment (persiaAziz) [inline tests/gold_tests/autest-site/trafficserver.test.ext]: Can ss be called only when there is a port binding problem?

**Comment by persiaAziz** [comment]
backport to 7.1? @dragon512 @zwoop

**Comment by dragon512** [comment]
We probably should

**Comment by zwoop** [comment]
Sure, please make a PR against 7.1.x, since this doesn't cherry-pick cleanly.

**Comment by zwoop** [comment]
@dragon512 ping on this? Can you make a 7.1.x PR ?

**Comment by dragon512** [comment]
@zwoop Yes. I will make it tomorrow

**Pull request merged**